### PR TITLE
[FEATURE] Add documentation for new task enhanceManifest and processor manifestEnhancer

### PR DESCRIPTION
**Pull request by flovogt**
Add documentation for new task enhanceManifest and processor manifestEnhancer

JIRA: CPOUI5FOUNDATION-831

### CODE REVIEW

File: docs/pages/Server.md
Comment (d3xter666) [inline docs/pages/Server.md:49]: Typo? 
```suggestion
Further file content transformations are executed:
```

Comment (flovogt) [reply docs/pages/Server.md]: thx. done

**Comment by d3xter666** [comment]
LGTM

File: docs/pages/Builder.md
Comment (matz3) [inline docs/pages/Builder.md:24]: Is this the actual order of execution?

File: docs/pages/Builder.md
Comment (matz3) [inline docs/pages/Builder.md:134]: The meaning of an empty array should also be explained here.

File: docs/pages/Builder.md
Comment (matz3) [inline docs/pages/Builder.md:90]: The minimum "_version" should be mentioned

File: docs/pages/Server.md
Comment (matz3) [inline docs/pages/Server.md:52]: not a valid link

File: docs/pages/Builder.md
Comment (matz3) [reply docs/pages/Builder.md:90]: It should also be mentioned that this behavior won't apply for files outside of the namespace of the manifest.json.

**Review by matz3 — DISMISSED** [review]

File: docs/pages/Builder.md
Comment (flovogt) [reply docs/pages/Builder.md:24]: Nope. I thought its sorted alphabetical but its not. So, I have moved the newly added row to the correct place.

File: docs/pages/Builder.md
Comment (flovogt) [reply docs/pages/Builder.md:134]: I would rather add a link to https://ui5.sap.com/#/topic/ec753bc539d748f689e3ac814e129563 instead of explaining runtime code here.

File: docs/pages/Builder.md
Comment (flovogt) [reply docs/pages/Builder.md:90]: done

File: docs/pages/Builder.md
Comment (KlattG) [inline docs/pages/Builder.md:94]:
```suggestion
This feature is only available for the following projects:
```

File: docs/pages/Builder.md
Comment (KlattG) [inline docs/pages/Builder.md:95]:
```suggestion
- The `_version` property in the `manifest.json` is set to `1.21.0` or higher
```

File: docs/pages/Builder.md
Comment (KlattG) [inline docs/pages/Builder.md:96]:
```suggestion
- The defined bundle is located inside the namespace of the `manifest.json`
```

**Review by KlattG — CHANGES_REQUESTED** [review]

**Review by KlattG — APPROVED** [review]
LGTM

**Review by RandomByte — CHANGES_REQUESTED** [review]

File: docs/pages/Builder.md
Comment (flovogt) [reply docs/pages/Builder.md]: done

File: docs/pages/Builder.md
Comment (flovogt) [reply docs/pages/Builder.md]: done

File: docs/pages/Builder.md
Comment (flovogt) [reply docs/pages/Builder.md]: done

File: docs/pages/Builder.md
Comment (flovogt) [reply docs/pages/Builder.md]: done

File: docs/pages/Builder.md
Comment (flovogt) [reply docs/pages/Builder.md]: done

File: docs/pages/Builder.md
Comment (flovogt) [reply docs/pages/Builder.md]: done

File: docs/pages/Builder.md
Comment (flovogt) [reply docs/pages/Builder.md]: done

File: docs/pages/Builder.md
Comment (RandomByte) [inline docs/pages/Builder.md:119]:
```suggestion
  }
}
```

Sorry for the nitpick 😄

File: docs/pages/Builder.md
Comment (RandomByte) [inline docs/pages/Builder.md:137]:
```suggestion
  }
}
```

**Review by RandomByte — CHANGES_REQUESTED** [review]

File: docs/pages/Builder.md
Comment (flovogt) [reply docs/pages/Builder.md]: you're right.... I even added another bracket

File: docs/pages/Builder.md
Comment (flovogt) [reply docs/pages/Builder.md]: you're right.... I even added another bracket

**Review by KlattG — APPROVED** [review]
LGTM

**Review by RandomByte — APPROVED** [review]

**Comment by flovogt** [comment]
Do not merge this PR until feature PRs are merged.

**Pull request merged**
